securedrop/journalist.py

```diff
@@ -195,8 +195,13 @@ def admin_add_user():
                     flash("That username is already in use",
                           "error")
                 else:
-                    flash("An error occurred saving this user to the database",
+                    flash("An error occurred "
+                          "saving this user to the database. "
+                          "Please check the application "
+                          "logs or inform your adminstrator.",
```

Comment:
Any of the routes with the `@admin_required` decorator are only accessible to admins, so you can get rid of the "or inform your administrator" here.